sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -39,7 +39,7 @@
    *
    * @param timestamp the timestamp
    */
-  public Breadcrumb(final @Nullable Date timestamp) {
+  public Breadcrumb(final @NotNull Date timestamp) {
```

Comment:
its recommended but not mandatory https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/
wondering if there's an use case where it makes sense not having a date

Reply:
Then we just need a decision. Right now when passing `null` as timestamp, `getTimestamp` throws NPE.

Reply:
Agree with `NotNull` here.

Sentry tries to be very lenient on what it receives but we can require this value from the SDK point of view.
One could always use the current time when instantiating. This is anyway more an internal API (should be IMHO) since users usually don't care about date unless writing some integration that's reading old logs (with dates) and creating crumbs.

Ideally we'd have overloads even such as: `Sentry.addBreadcrumb("message")` so there's no need to instantiate a `Breadcrumb` to begin with.